src/data/scripts/campaign/econ/industries/MPC_coronaResistStructure.kt

```diff
@@ -21,9 +21,9 @@ class MPC_coronaResistStructure: baseNikoIndustry() {
 
     override fun apply() {
         super.apply(true)
-
+        if (reapplying) return
         script = MPC_coronaResistStructureScript(market.primaryEntity, this)
-        script!!.start()
+        if (!script!!.started){script!!.start()}
```

Comment:
you misunderstand this code, my scripts have start() and startImpl()

start() is non-overridable and checks for started, if its already started startImpl is not called, meaning this optimization is already done within .start()

Reply:
well, if that part in particular isnt at fault, okay. though I know that the changes made to this script, whether a part or in whole, prevented the massive bogdown after opening the structure UI and closing.